airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/test/write/BasicFunctionalityIntegrationTest.kt

```diff
@@ -73,6 +75,14 @@ abstract class BasicFunctionalityIntegrationTest(
                         name = "test_stream",
                         data = """{"id": 5678}""",
                         emittedAtMs = 1234,
+                        changes =
```

Comment:
this maybe wants to be a json string for easier test-writing, curious what other folks think

Reply:
Also copilot will probably start one-shotting the structured stuff once there are a few examples in its context window.